src/tike/view.py

```diff
@@ -75,14 +75,12 @@ def plot_complex(Z, rmin=None, rmax=None, imin=None, imax=None):
     Takes parameters rmin, rmax, imin, imax to scale the ranges of the real
     and imaginary plots.
     """
-    plt.figure(dpi=128)
     plt.subplot(1, 2, 1)
     plt.imshow(Z.real, vmin=rmin, vmax=rmax)
     cb0 = plt.colorbar(orientation='horizontal')
     plt.subplot(1, 2, 2)
     plt.imshow(Z.imag, vmin=imin, vmax=imax)
     cb1 = plt.colorbar(orientation='horizontal')
-    plt.show()
 
 
 def plot_phase(Z, amin=None, amax=None):
```

Comment:
Function says plot_phase but you plot both amplitude and phase. Use plot_wave instead?